Added benchmarks for ReachSafety Arrays and ECA.

avritichauhan opened the pull request:
<!--
  Please describe your PR as usual.

  For submission of new verification tasks,
  keep the following checklist and make sure that all items are fullfilled.
  For other PRs, just remove it.
-->

- [x] programs added to new and [appropriately named](https://github.com/sosy-lab/sv-benchmarks/blob/master/CONTRIBUTING.md#directory-structure-and-names) directory
- [x] license present and [acceptable](https://github.com/sosy-lab/sv-benchmarks/blob/master/CONTRIBUTING.md#license) (either in separate file or as comment at beginning of program)
- [x] [contributed-by](https://github.com/sosy-lab/sv-benchmarks/blob/master/CONTRIBUTING.md#origin-description-and-attribution) present (either in README file or as comment at beginning of program)

- [x] programs added to a `.set` file of an existing category, or new sub-category established (if justified)
- [x] intended property matches the corresponding `.prp` file
- [x] expected answer in file names according to [convention](https://github.com/sosy-lab/sv-benchmarks/blob/master/CONTRIBUTING.md#properties)

<!-- For C programs: -->
- [x] [architecture](https://github.com/sosy-lab/sv-benchmarks/blob/master/CONTRIBUTING.md#architecture) (32 bit vs. 64 bit) matches the corresponding `.cfg` file
- [x] original sources present
- [x] [preprocessed](https://github.com/sosy-lab/sv-benchmarks/blob/master/CONTRIBUTING.md#preprocessing) files present
- [x] preprocessed files generated with correct architecture
- [x] [Makefile](https://github.com/sosy-lab/sv-benchmarks/blob/master/CONTRIBUTING.md#compile-checks) added with correct content and without overly broad suppression of warnings

kfriedberger commented on c/eca-programs/License.txt:
```diff
@@ -0,0 +1 @@
+../LICENSE.Apache-2.0.txt
```
Where does this license come from?
Are you an author or those tasks or did you ask them?
Are the programs taken from RERS2018?

dbeyer commented:
If the verification tasks are from RERS, then please name the folder accordingly (compare with the existing folder from 2012).
And also give the source from where the source files were downloaded.

I think Apache is good as license. I can ask Bernhard Steffen for confirmation (if they are really the RERS programs).

dbeyer commented:
I sent mail to Bernhard Steffen about the license in case those are the RERS programs.

dbeyer commented:
Bernhard replied that he is fine with Apache 2.
So please make sure that the license applies to the old and new programs (which is my understanding of the current PR situation).

dbeyer commented:
@avritichauhan Can you please finish this PR?

avritichauhan commented:
We have modified eca-rers2012 benchmarks to create the eca-programs benchmarks. 

Raised 2 separate PRs for ReachSafety Arrays(#600 ) and ECA (#601 ). Also, added licenses for old and new ECA benchmarks (#601 ). Hence, closing this PR.
